src/beignet/special/_faddeeva_w.py

```diff
@@ -1,14 +1,13 @@
+import math
+
 import torch
 from torch import Tensor
 
 
-def _voigt_v(x, y):
-    # assumes x >= 0, y >= 0
-
-    N = 11
-
-    # h = math.sqrt(math.pi / (N + 1))
-    h = 0.5116633539732443
+def _voigt_v(x, y, N: int = 11):
+    assert (x >= 0.0).all()
```

Comment:
My personal preference is to raise a `ValueError`. My reasoning is that Python’s behavior for a failed assert is to raise the less clear `AssertionError`. What do you think? Either way, we should probably document our policy around `assert` statements.

Reply:
Updated to raise a `ValueError`. I agree this is better. Asserts are also probably not good to use for validation since they get turned off completely by `python -O`